### DISCUSSION TITLE
Welcome to cookiecutter-hypermodern-python Discussions!

### DISCUSSION THREAD

**Discussion by cjolowicz** [cjolowicz/cookiecutter-hypermodern-python#698] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by oncleben31** [comment]
Hello I'm developing as a hobby, mainly in Python. All my recent activities on GitHub are driven by [Home Assistant](https://github.com/home-assistant/core) and the development of a [French speaking community for HA](https://github.com/hacf-fr).
I use this repository to learn and apply best practices on API communications Python modules. And I'm working on a [Cookiecutter template to help creating a custom  component for Home Assistant](https://github.com/oncleben31/cookiecutter-homeassistant-custom-component).

Thank you @cjolowicz for your work and your kindness in all your communications.

**Comment by Troubladore** [comment]
Hello!

I'm interested in putting together project templates to assist myself and others in developing on an open-source trading system called [NautilusTrader](https://github.com/nautechsystems/nautilus_trader). I'm new to Python, but old to programming, and would like to create a robust, reproducible experimentation space. I was really drawn to the hypermodern approach even before I found out there was a cookiecutter for it - now that I know it's here, I'm pretty elated!

Thank you @cjolowicz for the time you have invested in defining, socializing, and automating such a robust project model for Python!
